tests/h/services/annotation_stats_test.py

```diff
@@ -69,36 +72,93 @@ def test_user_annotation_counts_ignores_deleted_annotations(self, svc, factories
         assert results['group'] == 0
         assert results['total'] == 0
 
-    def test_annotation_count_returns_count_of_shared_annotations_for_group(self, svc, factories):
+    def test_annotation_count_returns_count_of_shared_annotations_for_group(self, svc, Annotation):
         pubid = 'abc123'
-        for i in range(3):
-            factories.Annotation(groupid=pubid, shared=True)
+        Annotation(groupid="wrong_grp", shared=True)
         for i in range(2):
```

Comment:
I felt we were creating an unnecessary amount of annotations here so I decreased the numbers a little.